packages/dropdowns/src/styled/field/StyledMultiselect.ts

```diff
@@ -0,0 +1,117 @@
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, getLineHeight, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { StyledInput } from './StyledInput';
+import { SelectWrapper, IStyledSelectProps } from './StyledSelect';
+
+const sizeStyles = (props: IStyledSelectProps & ThemeProps<DefaultTheme>) => {
+  const tagHeight = props.isCompact ? 20 : 32;
+  const inputHeight = props.isCompact ? 32 : 40;
+  const tagMargin = props.theme.space.base / 2;
+  const inputBorder = props.isBare ? 0 : stripUnit(props.theme.borderWidths.sm);
+  const verticalPadding = `${(inputHeight - tagHeight - inputBorder * 2 - tagMargin * 2) / 2}px`;
+  const horizontalPadding = `${props.theme.space.base * 2}px`;
+
+  return css`
+    padding-top: ${verticalPadding};
+    padding-bottom: ${verticalPadding};
+    /* stylelint-disable-next-line property-no-unknown */
+    padding-${props.theme.rtl ? 'right' : 'left'}: ${horizontalPadding};
+  `;
+};
+
+export const StyledMultiSelect = styled(SelectWrapper)`
+  display: flex;
+  flex-wrap: wrap;
+
+  ${props => sizeStyles(props)};
+`;
+
+StyledMultiSelect.defaultProps = {
+  theme: DEFAULT_THEME
+};
+
+interface IStyledItemWrapperProps {
+  isCompact?: boolean;
+}
+
+export const StyledItemWrapper = styled.div<IStyledItemWrapperProps>`
+  margin: ${props => props.theme.space.base / 2}px;
+  max-width: 100%;
+  max-height: ${props =>
+    props.isCompact ? props.theme.space.base * 5 : props.theme.space.base * 8}px;
+  line-height: 0;
+`;
+
+StyledItemWrapper.defaultProps = {
+  theme: DEFAULT_THEME
+};
+
+interface IStyledMultiselectInputProps {
+  isCompact: boolean;
+  isVisible: boolean;
+}
+
+const hiddenStyling = css`
+  opacity: 0;
+  margin: 0;
+  width: 0;
+  min-width: 0;
+  height: 0;
+  min-height: 0;
+`;
+
+export const StyledMultiselectInput = styled(StyledInput)<IStyledMultiselectInputProps>`
+  flex-basis: ${props => props.theme.space.base * 15}px;
+  flex-grow: 1;
+  margin: ${props => props.theme.space.base / 2}px;
+  width: inherit;
+  min-width: ${props => props.theme.space.base * 15}px;
+  line-height: ${props =>
+    getLineHeight(
+      props.isCompact ? props.theme.space.base * 5 : props.theme.space.base * 8,
+      props.theme.fontSizes.md
+    )};
+
+  ${props => !props.isVisible && hiddenStyling}
+`;
+
+StyledMultiselectInput.defaultProps = {
+  theme: DEFAULT_THEME
+};
+
+export const StyledMoreAnchor = styled.div`
+  display: flex;
+  align-items: center;
+  margin: ${props => props.theme.space.base / 2}px;
+  border: none;
+  border-radius: 0;
+  background-color: transparent;
+  cursor: pointer;
+  padding: 0;
+  min-width: 0;
+  min-height: 1em;
+  vertical-align: baseline;
+  text-decoration: none;
+  user-select: none;
+  line-height: 1em;
+  white-space: normal;
```

Comment:
`white-space` should be `no-wrap` and configure the anchor so that overflow is truncated. Shouldn't be seeing this: 
![Screen Shot 2020-02-07 at 1 07 56 PM](https://user-images.githubusercontent.com/143773/74065940-e4326f80-49aa-11ea-9ca2-6407e49fd631.png)

Also, the anchor shouldn't be overflowing the end padding of the container.

Reply:
I'm struggling to get the existing Tag overflow styling to work correctly with our flex implementation. Could you take a closer look at this with the most recent commit?

Reply:
It's Tag `box-sizing` 😩(along with a flex `min-width` issue in this PR). I'll get a fix done for Tags under separate cover – also addressing the Avatar push-off we see over there:

<img width="902" alt="Screen Shot 2020-02-08 at 12 45 58 PM" src="https://user-images.githubusercontent.com/143773/74091815-0a224780-4a71-11ea-8e9f-3b35f5cb8dc2.png">

I'll follow-up with comments here. The fixes _might_ allow us to simplify the wrapping DOM structure.

Reply:
Addressing these issues with your new commit and update `min-width` values.